contracts/src/v0.8/dev/KeeperRegistryBase.sol

```diff
@@ -132,6 +136,7 @@ abstract contract KeeperRegistryBase is ConfirmedOwner, ExecutionPrevention, Ree
     bytes performData
   );
   event UpkeepCanceled(uint256 indexed id, uint64 indexed atBlockHeight);
+  event UpkeepPaused(uint256 indexed id, bool paused);
```

Comment:
could we make this two events? UpkeepPaused and UpkeepActivated?

Reply:
Done